### PR TITLE
Release 1.12.0 doc update

### PR DESCRIPTION
**Pull request by therahulbhati**
Signed-off-by: therahulbhati <rjbhati009@gmail.com>

### CODE REVIEW

File: docs/content/en/docs/releases/1.12.0.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Features
+
+#### Support for Keda 2.0
+
+From this release, the message queue trigger of `--mqtkind keda`, supports `Keda 2.0` we won't be supporting `Keda 1.5`. Checkout out [compatibility page](https://docs.fission.io/docs/compatibility) for more details.
+
+#### Multi-Architecture Build
+
+Thanks to [cerebrate](https://github.com/cerebrateNow), now you can deploy [fission on ARM architecture](https://github.com/fission/fission/pull/1891).
```
Comment (RealHarshThakur) [inline docs/content/en/docs/releases/1.12.0.md:15]: Do we actually have mutliarch images in this release?

File: docs/content/en/docs/compatibility/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+description: >
+  Fission Compatibility with Keda and Keda Connectors
+---
+
+| Fission version | Keda version | Keda Connectors image tag |
+| --------------- | ------------ | ------------------------- |
+| 1.11.2          | 1.5          | v0.1                      |
+| 1.12.0          | 2.0          | v0.5                      |
+
+All Keda Connector images can be used with both fission version except the Kafka connector, which has some breaking changes.
```
Comment (vishal-biyani) [inline docs/content/en/docs/compatibility/_index.md:13]: Should we also add a few lines about the fact that Environment images will follow a different versioning scheme going forward?

**Review by vishal-biyani — CHANGES_REQUESTED** [review]
One small comment on env - but looks good otherwise.

File: docs/content/en/docs/compatibility/_index.md (hunk 2 above)
Comment (therahulbhati) [reply docs/content/en/docs/compatibility/_index.md:13]: Yes, will add it

**Pull request merged**